### DISCUSSION TITLE
The API isn't transcribing filler words.

### DISCUSSION THREAD

**Discussion by aprobert-li** [deepgram/community#916] [General help] [answered]
Both in the playground and in my code with filler_words set to true, filler words are missed in the transcription.  Has anyone had luck with this working for them?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment]
@aprobert-li  could you provide  what was suggested by our bot ☝️  above so we can troubleshoot?

**Reply by aprobert-li** [reply]
Yeah I can provide a sample video file and the transcript so you can see.  It's even in the API playground that it's not transcribing filler words.  Here's an example.  The transcript does show one "um".  Compare that with the audio from the recording.
<img width="645" alt="Screen Shot 2024-09-10 at 8 47 14 PM" src="https://github.com/user-attachments/assets/80294baf-5d8b-4876-86eb-2e80a600c687">

https://github.com/user-attachments/assets/3ad45c53-6509-4083-94c8-330f09e350fc

**Comment by aprobert-li** [comment]
@jpvajda I just wanted to follow-up on this.  Thanks

**Comment by jpvajda** [comment]
@aprobert-li  when you run something the Playground it will return a `request ID`, can you provide that please? You can find it under JSON > Metadata.  You can just copy and paste it. 


![Screenshot 2024-09-13 at 12 59 38 PM](https://github.com/user-attachments/assets/22544440-03eb-4206-9e74-c48d801f6f66)


 The video is helpful  but it doesn't really allow us to troubleshoot the audio. Also providing the actual audio file will help too if you can do give us that.

If it's not sensative audio, just drop in this thread, or you can email it to `devrel@deepgram.com` and mention this `GitHub Discussion #919`

**Reply by aprobert-li** [reply]
The request id is:  "b48267da-4c71-4de7-8677-6e4087a60293".

I attached the audio sample in my earlier response.  It's the deepgram-sample.mp4.

Thanks!

**Comment by jpvajda** [comment, marked as answer]
@aprobert-li  Here's what i see on our end:

`Warning: A video file was sent to Deepgram. This is not an issue 99% of the time, but occasionally Deepgram has trouble with certain video formats.`

Suggestion: Try rerunning the request with just the audio to see if any issues are resolved.  It's possible the mp4 format is just making the model miss some filler words.
